CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+14.1.0
```

Comment:
Does the .NET library have the concept of _latest_, and if so and someone uses that, will their code break with this release?

I think the answer is "no", but could you confirm?

Reply:
No, we don't. So all good here.

Reply:
it does have the concept of latest, but I don't think this release will break the code of users depending on our latest version, since this adds support for V12 and does not remove support for V10 and V11 or impact it in any other way.